examples/babel.server.js

```diff
@@ -0,0 +1,5 @@
+require("babel/register")({
```

Comment:
please add the license information at the top of each example file (see app.jsx)